draco/analysis/ringmapmaker.py

```diff
@@ -230,7 +231,7 @@ def process(self, gstream):
                 vmax = nsmax / wv
 
             if self.weight == "inverse_variance":
-                gw = gsw[:, lfi].copy()
+                gw = gsw[:, lfi].view(np.ndarray).copy()
```

Comment:
Should the `view()` occur before the slice?

Reply:
I believe that both `.local_array` and moving the view before the slice is old-code compatible!